Source/WebCore/page/DOMWindow.idl

```diff
@@ -84,8 +84,6 @@
     // the user agent
     readonly attribute Navigator navigator;
     [Replaceable, ImplementedAs=navigator] readonly attribute Navigator clientInformation;
-    // FIXME: This is specified to be [SecureContext]
-    [EnabledByQuirk=shouldEnableApplicationCache] readonly attribute DOMApplicationCache applicationCache;
```

Comment:
Can we remove the quirk as well?

Reply:
No, it guards DOMApplicationCache too. And removing that I probably have to leave for @szewai.